client/src/components/Dashboard/AddItem.js

```diff
@@ -0,0 +1,135 @@
+import React, { useState, useEffect } from 'react';
+import {
+  makeStyles,
+  MuiThemeProvider,
+  createMuiTheme,
+} from '@material-ui/core/styles';
+import Button from '@material-ui/core/Button';
+import Input from '@material-ui/core/Input';
+import MenuItem from '@material-ui/core/MenuItem';
+import Select from '@material-ui/core/Select';
+import Typography from '@material-ui/core/Typography';
+
+const addItemTheme = createMuiTheme({
+  palette: {
+    primary: { main: '#DF1B1B' },
+  },
+});
+
+const mockData = [
+  { title: 'Scented Candles' },
+  { title: 'Towels' },
+  { title: 'Kale' },
+];
+
+const useStyles = makeStyles(theme => ({
+  container: {
+    display: 'flex',
+    flexWrap: 'wrap',
+    justifyContent: 'center',
+  },
+  label: {
+    margin: 50,
+  },
+  input: {
+    width: 400,
+    height: 30,
+    maxWidth: '100%',
+    backgroundColor: 'white',
+    fontFamily: theme.typography.fontFamily,
+    fontSize: '1.6rem',
+    padding: '15px 25px',
+    borderRadius: '28px 0 0 28px',
+    boxShadow: theme.boxShadowTheme,
+  },
+  selectContainer: {
+    display: 'flex',
+    justifyContent: 'space-around',
+    width: 320,
+    maxWidth: '100%',
+    backgroundColor: 'white',
+    padding: 5,
+    borderRadius: '0 28px 28px 0',
+    boxShadow: theme.boxShadowTheme,
+  },
+  select: {
+    width: 180,
+    maxWidth: '100%',
+    padding: 10,
+    fontFamily: theme.typography.fontFamily,
+    fontSize: '1.6rem',
+  },
+  addButton: {
+    fontSize: '1.6rem',
+    borderRadius: 24,
+    width: 140,
+    position: 'relative',
+    left: '140',
+  },
+  menuItem: {
+    fontSize: '1.6rem',
+  },
+}));
+
+export default function AddItem() {
+  const classes = useStyles();
```

Comment:
Changing AddItem() to AddItem(props) and useStyles() to useStyles(props) provide it with the theme object.